react/features/participants-pane/components/web/MeetingParticipantItem.js

```diff
@@ -173,34 +173,44 @@ function MeetingParticipantItem({
     const [ hasAudioLevels, setHasAudioLevel ] = useState(false);
     const [ registeredEvent, setRegisteredEvent ] = useState(false);
 
-    const _updateAudioLevel = level => {
+    const _updateAudioLevel = useCallback(level => {
         const audioLevel = typeof level === 'number' && !isNaN(level)
             ? level : 0;
 
         setHasAudioLevel(audioLevel > 0.009);
-    };
+    }, []);
 
     useEffect(() => {
         if (_audioTrack && !registeredEvent) {
             const { jitsiTrack } = _audioTrack;
 
-            jitsiTrack && jitsiTrack.on(JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED, _updateAudioLevel);
-            setRegisteredEvent(true);
+            if (jitsiTrack) {
+                jitsiTrack.on(JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED, _updateAudioLevel);
+                setRegisteredEvent(true);
+            }
         }
+
+        return () => {
+            if (_audioTrack && registeredEvent) {
+                const { jitsiTrack } = _audioTrack;
+
+                jitsiTrack && jitsiTrack.off(JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED, _updateAudioLevel);
+            }
+        };
     }, [ _audioTrack ]);
 
-    const _getAudioMediaState = useMemo(() => {
+    const _getAudioMediaState = () => {
```

Comment:
Do we need a function for this? Wouldn't a simple variable suffice?

Reply:
Yes, it can be a variable. Changed